lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -68,39 +68,6 @@
     <div class="request-response-error"></div>
 </fieldset>
 
-<!-- Uploading a CSV file of bulk enrollment. -->
```

Comment:
Why we are removing the bulk enrollment feature?

Reply:
@rehan99000 accidentally removed it and now added it back: https://github.com/edly-io/edx-platform/pull/198#discussion_r565083293

Reply:
it was a mistake. I have reverted the changes